lib/cloudregister/registerutils.py

```diff
@@ -246,7 +246,7 @@ def fetch_smt_data(cfg, proxies, quiet=False):
         except Exception as e:
             logging.error('=' * 20)
             logging.error(str(e))
-            logging.error('Unable to obtain SMT server information, exiting')
+            logging.error('Unable to obtain update server information, exiting')
```

Comment:
seems this change is unrelated to this PR or to make network message less alarming ?

Reply:
Yes it is, stumbled over it and having a PR to change 1 word is a bit overkill on process.